_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -502,7 +509,32 @@ public static function get_plans( $request ) {
 		}
 
 		return $data;
-	}
+    }
+
+    public static function get_products( $request ) {
+        $wpcom_request = Client::wpcom_json_api_request_as_user(
+            '/products?_locale=' . get_user_locale(),
+            'v1.1',
+			array(
+				'method'  => 'GET',
+				'headers' => array(
+					'X-Forwarded-For' => Jetpack::current_user_ip( true ),
+				),
+			),
+	        null,
+	        'rest'
+        );
+
+	    $body = json_decode( wp_remote_retrieve_body( $wpcom_request ) );
+	    if ( 200 === wp_remote_retrieve_response_code( $wpcom_request ) ) {
+		    $data = $body;
+	    } else {
+		    // something went wrong so we'll just return the response without caching
```

Comment:
Where does the caching happen?

Reply:
There was no caching. I had copied it from the other calls in this file assuming that it was part of a framework, but on further investigation I found it was just something left over from old code that never got removed.